scripts/read_spectrum.R

```diff
@@ -294,6 +304,7 @@ prepare_hc_leapfrog_projp <- function(pjnz, params, pop_1){
   v$prop_lt200 <- colSums(wlhiv_cd4[5:7,]) / colSums(wlhiv_cd4)
 
   v$hc_age_coarse <- rep(c(1,2,3), each = 5)
+  v$hc_age_coarse_cd4 <- c(0,0,0,1,1,rep(2,10))
```

Comment:
Might code this as `rep(0:2, times = c(3, 2, 10))`. But inconsequential.

Reply:
Sure